### DISCUSSION TITLE
dissapearing shadows with gradient mesh shader

### DISCUSSION THREAD

**Discussion by renato3xl** [bnpr/Malt#172] [Q&A] [answered]
Hi! i'm new to all of this, (specially malt and github) so sorry if i say something to obvious or ask something that has been asked 200 times.

I'm an animator and i have been working in maya for 20 year, and now i have moved into blender, and i really like what you can do with Malt, i have done some animations already but not i find some problems.
This one that is probably the easiest to fix is that i use the override function on the scene because i render all in layers and make the final look in compositing.
I want to take a shadow pass and i found that the gradient mesh shader does exacly what i need, BUT, depending on the possition of the camera the shadows disspear, the far i go the worst.

Any clue on how to fix this? i can try to upload a video or image if my problem is not clear enough.

Thanks so much! i hope i will learn a lot with you!

**Comment by pragma37** [comment, marked as answer]
Hi! You're welcome!

> sorry if i say something to obvious or ask something that has been asked 200 times.

No worries. The engine is still quite new, so there's still plenty of lack of documentation and tutorials.

It sounds that you may want to increase the Sun shadows *Max Distance* in the Scene settings:

![imagen](https://user-images.githubusercontent.com/36610999/143044261-8860517e-74f9-45d4-90ab-dfa3ab42a6fd.png)

Keep in mind that if you increase the Max Distance you may also want to increase the shadow Resolution and/or Cascades Count.
